Makefile

```diff
@@ -109,12 +110,17 @@ third-party-test-venv: FORCE
 third-party-chpldoc-venv: FORCE
 	cd third-party && $(MAKE) chpldoc-venv
 
+third-party-scspell-venv: FORCE
```

Comment:
I'd be tempted to view this as a developer-only feature, myself - I think that a user should be responsible for their own spelling errors and that we shouldn't make spell-checking a service we provide.